redwood-lazylayout-widget/src/commonTest/kotlin/app/cash/redwood/lazylayout/widget/FakeUpdateProcessor.kt

```diff
@@ -95,7 +96,8 @@ class FakeUpdateProcessor : LazyListUpdateProcessor<FakeUpdateProcessor.StringCe
         scrollWindowOffset--
       } else if (index < scrollWindowLimit) {
         // Delete a cell from the scroll window. We'll replenish the window below.
-        scrollWindowCells.removeAt(index - scrollWindowOffset).binding.unbind()
+        val removed = scrollWindowCells.removeAt(index - scrollWindowOffset)
+        require(!removed.binding.isPlaceholder && removed.content == null)
```

Comment:
This was hiding the bug. . . the fake processor was unbinding synchronously anyway. With this change it now asserts that it’s unbound.